.schema/config.schema.json

```diff
@@ -1127,6 +1127,11 @@
             "enabled": {
               "$ref": "#/definitions/handlerSwitch"
             },
+            "key": {
+              "type": "string",
+              "title": "Cache key",
+              "description": "Define custom cache key, i.e '{{ print .Subject }}'. All properties must be part of AuthenticationSession."
```

Comment:
One difficulty with this is that a singular typo (e.g. `{{ print .Subject}` instead of `{{ print .Subject }}`) can lead to catastrophic cache confusion. So it's very easy to get this wrong because we don't know if this is a literal string or a template string.

Reply:
Let me give this some further thought what can be done here while retaining flexibility. As of now the following assertions are in place to reduce likelyhood of cache confusion.

1. `subject` must match cached `subject` of `AuthenticationSession`.
2. Configured URL of `hydrator` is part of the composite cache key, i.e. no cache key of another configured `hydrator` URL will overlap.
3. Rule ID will be part of the of the composite cache key (as per this PR), ensuring cache key scope is limited to rule.

Reply:
I believe the existing method is as accurate as it can be.